src/schema/v2/partner.ts

```diff
@@ -213,7 +242,9 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
         resolve: ({ pre_qualify }) => pre_qualify,
       },
       locations: {
-        type: new GraphQLList(Location.type),
+        type: new GraphQLList(LocationType),
+        description:
```

Comment:
There's a first-class way to deprecate a field (search for `deprecationReason` in MP).

Reply:
Additionally we have a `deprecate` function which generates standardized messages, including a suggestion for what field to use instead.